pkg/config/config_template.yaml

```diff
@@ -921,6 +921,15 @@ api_key:
     #       in_speed: 50
     #       out_speed: 25
 
+    ## @param tags - list of strings - optional
+    ## A list of tags to attach to every metric and service check of all devices discovered in the subnet.
+    ##
+    ## Learn more about tagging at https://docs.datadoghq.com/tagging
+    #
+    # tags:
+    #   - <KEY_1>:<VALUE_1>
+    #   - <KEY_2>:<VALUE_2>
```

Comment:
Not a blocker but since tags are pretty important from a UX perspective, I wonder if they should be up higher in its parent `configs` group.

Reply:
If you mean having tags at `snmp_listener.tags` level, I think it might be redundant with root level `tags` in datadog.yaml since they will both be applied globally.

Reply:
Ah yeah, I meant that this should move up in the same section (`configs`) not that it needs moving up into the parent

Reply:
I'm not sure to fully understand, could you give a concrete example config? Right now `snmp_listener.configs` is a list.

Reply:
I see, sorry I though you meant moving it to a higher level in the config path.

Done here https://github.com/DataDog/datadog-agent/pull/16019/commits/ea7a7a3d8913f18a0471b559966e53203b3cd684